<!DOCTYPE html>
<html>
<head>
<title>Running a Simple Search Using Tide and Percolator</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0" />
<link rel="stylesheet" type="text/css" href="../styles.css">
<script type="text/javascript">
  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-26136956-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();
</script>
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.3/jquery.min.js"></script>
<script type="text/javascript">
	// Main Menu
	$( document ).ready(function() {
			var pull 		= $('.btn');
				menu 		= $('nav ul');
				menuHeight	= menu.height();

			$(pull).on('click', function(e) {
				e.preventDefault();
				menu.slideToggle();
			});

			$(window).resize(function(){
	    		var w = $(window).width();
	    		if(w > 320 && menu.is(':hidden')) {
	    			menu.removeAttr('style');
	    		} 
			});
	});
</script>
</head>
<body>
	<div class="page-wrap">
	 	<nav>
			<div class="btn">
		    </div>
		    <img src="../images/crux-logo.png" id="logo"></a>
			<ul id="navitems">
			    <li><a href="../index.html">Home</a></li>
			    <li><a href="../download.html">Download</a></li>
			    <li><a href="../fileformats.html">File Formats</a></li>
		        <li><a href="http://groups.google.com/group/crux-users">Contact</a></li> <!--Link to google support board-->
		    </ul>
		</nav>
		<div id="content">
		<!-- START CONTENT -->
		<h2>Running a Simple Search Using Tide and Percolator</h2>

		<p>
		Now that you have your environment <a href="gettingstarted.html">set
		up</a> and the two input files in your working directory, you can
		conduct the search. The search process compares each spectrum
		in <span class=file>demo.ms2</span> to peptides (subsequences of the
		proteins) in fasta files provided in a
		dirctory, <span class=file>yeast-index/</span>.  Peptides whose
		precursor mass is close to that of the observed spectrum are scored
		against that spectrum, and the top scores are reported in the output.
		To conduct the search, we first create a peptide index
		using <code>tide-index</code> and then execute the search
		using <code>tide-search</code>.</p>

		<ol>
		<li>
		<div class=code>
		<code>$ crux tide-index small-yeast.fasta yeast-index </code>
		</div>

		<p>
		While generating the peptide index, you will see output like this:</p>

		<pre>
INFO: CPU: pyrrolysine.gs.washington.edu
INFO: Crux version: 2.1
INFO: Fri Feb  5 11:24:02 PST 2016
COMMAND LINE: ./crux tide-index small-yeast.fasta yeast-index
INFO: Running tide-index...
INFO: Writing results to output directory 'yeast-index'.
INFO: Reading small-yeast.fasta and computing unmodified peptides...
INFO: Writing decoy fasta...
INFO: Reading proteins
INFO: Precomputing theoretical spectra...
INFO: Elapsed time: 0.0973 s
INFO: Finished crux tide-index.
INFO: Return Code:0
		</pre>

		<p>
		This command produces the peptide index in <code>yeast-index</code>
		and also produces a directory <code>crux-output</code> containing 
		the following files:
		<ol>

		<li>
		tide-index.decoy.fasta &ndash; a set of decoy proteins, derived from
		the proteins in the input set,</li>

		<li>
		tide-search.params.txt &ndash; a record of all the parameters used in
		the search, and</li>

		<li>tide-search.log.txt &ndash; a log file containing a copy of all
		  the messages printed to the screen during the search.</li>
		</ol>

		<p>
		Now you can run this command:</p></li>

		<div class=code>
		<li><code>$ crux tide-search --compute-sp T demo.ms2 yeast-index</code>
		</div>

		<p>While the search is running, you will see output like this:</p>

		<pre>
INFO: CPU: pyrrolysine.gs.washington.edu
INFO: Crux version: 2.1
INFO: Fri Feb  5 11:24:23 PST 2016
COMMAND LINE: ./crux tide-search --compute-sp T demo.ms2 yeast-index
INFO: Running tide-search...
INFO: Reading index yeast-index
INFO: Converting demo.ms2 to spectrumrecords format
INFO: Reading spectra file crux-output/demo.ms2.spectrumrecords.tmp
INFO: Sorting spectra
INFO: Running search
INFO: Time per spectrum-charge combination: 0.002318 s.
INFO: Average number of candidates per spectrum-charge combination: 15.204820 
INFO: Elapsed time: 0.389 s
INFO: Finished crux tide-search.
INFO: Return Code:0
		</pre>

		<p>
		The <span class=file>crux-output</span> directory now contains four new files
		containing the search results:

		<ol>

		<li>
		tide-search.target.txt &ndash; search results
		in <a href="../file-formats/txt-format.html">tab-delimited format</a>.</li>

		<li>
		tide-search.decoy.txt &ndash; search results from a decoy database
		in <a href="../file-formats/txt-format.html">tab-delimited format</a>.</li>

		<li>
		tide-search.params.txt &ndash; a record of all the parameters used in
		the search.</li>

		<li>tide-search.log.txt &ndash; a log file containing a copy of all
		  the messages printed to the screen during the search.</li>
		</ol>

		</p>
		</li>

		<p>
		  Note that the peptide-spectrum matches (PSMs) in the
		  tide-search.target.txt are sorted by the precursor m/z value
		  associated with the spectrum. If you want to see which PSMs got the
		  highest XCorr scores, you can do so like this:</p>

		<div class=code>
		<li><code>$ crux sort-by-column --column-type real --ascending T crux-output/tide-search.target.txt "xcorr score" > crux-output/tide-search.target.sort.txt</code>
		</div>

		<p>
		  The first lines of the resulting sorted output file should look like this:</p>

		<table class="searchtable">
		<td><b>file</b></td><td><b>scan</b></td><td><b>charge</b></td><td><b>spectrum precursor m/z</b></td><td><b>spectrum neutral mass</b></td><td><b>peptide mass</b></td><td><b>delta_cn</b></td><td><b>sp score</b></td><td><b>sp rank</b></td><td><b>xcorr score</b></td><td><b>xcorr rank</b></td><td><b>b/y ions matched</b></td><td><b>b/y ions total</b></td><td><b>distinct matches/spectrum</b></td><td><b>sequence</b></td><td><b>cleavage type</b></td><td><b>protein id</b></td><td><b>flanking aa</b></td></tr>
		<tr>
		<td>demo.ms2</td><td>85</td><td>3</td><td>497.618</td><td>1489.83</td><td>1488.82</td><td>0.936932</td><td>2430.56</td><td>1</td><td>5.20757</td><td>1</td><td>27</td><td>48</td><td>3</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W(27)</td><td>RN</td></tr>
		<tr>
		<td>demo.ms2</td><td>118</td><td>3</td><td>1031.94</td><td>3092.8</td><td>3093.41</td><td>0.949698</td><td>2383.16</td><td>1</td><td>4.76204</td><td>1</td><td>44</td><td>108</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C(500)</td><td>KD</td></tr>
		<tr>
		<td>demo.ms2</td><td>156</td><td>3</td><td>1032.44</td><td>3094.3</td><td>3093.41</td><td>0.938358</td><td>1929.77</td><td>1</td><td>4.7476</td><td>1</td><td>43</td><td>108</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C(500)</td><td>KD</td></tr>
		<tr>
		<td>demo.ms2</td><td>18</td><td>3</td><td>1032.4</td><td>3094.18</td><td>3093.41</td><td>0.902056</td><td>1732.23</td><td>1</td><td>4.48933</td><td>1</td><td>40</td><td>108</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C(500)</td><td>KD</td></tr>
		<tr>
		<td>demo.ms2</td><td>11</td><td>2</td><td>745.269</td><td>1488.52</td><td>1488.82</td><td>0.822851</td><td>2816.6</td><td>1</td><td>4.44855</td><td>1</td><td>21</td><td>24</td><td>4</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W(27)</td><td>RN</td></tr>
		<tr>
		<td>demo.ms2</td><td>53</td><td>2</td><td>745.749</td><td>1489.48</td><td>1488.82</td><td>0.935648</td><td>2797.78</td><td>1</td><td>4.39828</td><td>1</td><td>21</td><td>24</td><td>3</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W(27)</td><td>RN</td></tr>
		<tr>
		<td>demo.ms2</td><td>38</td><td>3</td><td>1032.39</td><td>3094.15</td><td>3093.41</td><td>0.975741</td><td>1460.28</td><td>1</td><td>4.39627</td><td>1</td><td>37</td><td>108</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C(500)</td><td>KD</td></tr>
		<tr>
		<td>demo.ms2</td><td>42</td><td>3</td><td>1032.26</td><td>3093.76</td><td>3093.41</td><td>1.0216</td><td>1509.47</td><td>1</td><td>4.35676</td><td>1</td><td>38</td><td>108</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C(500)</td><td>KD</td></tr>
		<tr>
		<td>demo.ms2</td><td>111</td><td>3</td><td>1033.02</td><td>3096.04</td><td>3093.41</td><td>0.952727</td><td>1366.71</td><td>1</td><td>4.3498</td><td>1</td><td>36</td><td>108</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C(500)</td><td>KD</td></tr>
		<tr>
		<td>demo.ms2</td><td>62</td><td>2</td><td>745.859</td><td>1489.7</td><td>1488.82</td><td>0.913711</td><td>2766.65</td><td>1</td><td>4.34057</td><td>1</td><td>21</td><td>24</td><td>3</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W(27)</td><td>RN</td></tr>
		</table>


		<p>
		The final step is to post-process the search results using Percolator.
		Each spectrum has been compared to many peptides and we would like to
		return only the best match for each spectrum.  We also expect that
		some fraction of the spectra will not be identifiable as peptides (due
		to chemical noise, multiple peptides co-eluting, poor fragmentation,
		etc.).  The analysis step filters out those spectra and ranks the
		matches by quality.
		</p></li>

		<li>
		<div class=code><code>
		$ crux percolator --test-fdr 0.1 crux-output/tide-search.target.txt</code>
		</div>

		<p>While the analysis is running, you will see output like this
		</p>

		<pre>
INFO: CPU: pyrrolysine.gs.washington.edu
INFO: Crux version: 3.1-ccddffef5395314ce3a224ad5b34167229341653
INFO: Tue Dec 12 15:47:13 PST 2017
COMMAND LINE: ./crux percolator --test-fdr 0.1 crux-output/tide-search.target.txt
INFO: Beginning percolator.
INFO: Reading file crux-output/tide-search.target.txt
INFO: Converting input to pin format.
INFO: Parsing crux-output/tide-search.target.txt
INFO: Assigning index 0 to demo.ms2.
INFO: Parsing crux-output/tide-search.decoy.txt
INFO: There are 690 target matches and 690 decoys
INFO: Maximum observed charge is 3.
INFO: File conversion complete.
INFO: Percolator version 3.01.nightly-22-2f4e2a6, Build Date Nov 22 2017 10:07:29
INFO: Copyright (c) 2006-9 University of Washington. All rights reserved.
INFO: Written by Lukas Käll (lukall@u.washington.edu) in the
INFO: Department of Genome Sciences at the University of Washington.
INFO: Issued command:
INFO: percolator --results-peptides crux-output/percolator.target.peptides.txt --decoy-results-peptides crux-output/percolator.decoy.peptides.txt --results-psms crux-output/percolator.target.psms.txt --decoy-results-psms crux-output/percolator.decoy.psms.txt --verbose 2 --protein-decoy-pattern decoy_ --seed 1 --subset-max-train 0 --trainFDR 0.01 --testFDR 0.1 --maxiter 10 --search-input auto --no-schema-validation --protein-enzyme trypsin --post-processing-tdc crux-output/make-pin.pin
INFO: Started Tue Dec 12 15:47:13 2017
INFO: Hyperparameters: selectionFdr=0.01, Cpos=0, Cneg=0, maxNiter=10
INFO: Reading tab-delimited input from datafile crux-output/make-pin.pin
INFO: Features:
INFO: lnrSp deltLCn deltCn XCorr Sp IonFrac PepLen Charge1 Charge2 Charge3 enzN enzC enzInt lnNumDSP dM absdM 
INFO: Found 1380 PSMs
INFO: Separate target and decoy search inputs detected, using target-decoy competition on Percolator scores.
INFO: Train/test set contains 690 positives and 690 negatives, size ratio=1 and pi0=1
INFO: Selecting Cpos by cross-validation.
INFO: Selecting Cneg by cross-validation.
INFO: Split 1:	Selected feature 2 as initial search direction. Could separate 44 training set positives in that direction.
INFO: Split 2:	Selected feature 5 as initial search direction. Could separate 41 training set positives in that direction.
INFO: Split 3:	Selected feature 2 as initial search direction. Could separate 32 training set positives in that direction.
INFO: Found 39 test set positives with q<0.1 in initial direction
INFO: Reading in data and feature calculation took 0.077138 cpu seconds or 0 seconds wall clock time.
INFO: ---Training with Cpos selected by cross validation, Cneg selected by cross validation, fdr=0.01
INFO: Iteration 1:	Estimated 71 PSMs with q<0.1
INFO: Iteration 2:	Estimated 73 PSMs with q<0.1
INFO: Iteration 3:	Estimated 73 PSMs with q<0.1
INFO: Iteration 4:	Estimated 73 PSMs with q<0.1
INFO: Iteration 5:	Estimated 73 PSMs with q<0.1
INFO: Iteration 6:	Estimated 73 PSMs with q<0.1
INFO: Iteration 7:	Estimated 73 PSMs with q<0.1
INFO: Iteration 8:	Estimated 73 PSMs with q<0.1
INFO: Iteration 9:	Estimated 73 PSMs with q<0.1
INFO: Iteration 10:	Estimated 73 PSMs with q<0.1
INFO: Learned normalized SVM weights for the 3 cross-validation splits:
INFO:  Split1	 Split2	 Split3	FeatureName
INFO:  0.3140	 0.0590	 0.2375	lnrSp
INFO:  0.1862	-0.1121	 0.1878	deltLCn
INFO:  0.1340	 0.0416	 0.3738	deltCn
INFO:  0.0642	 0.3209	 0.6567	XCorr
INFO:  0.7780	 0.3290	 1.2113	Sp
INFO:  0.2765	 0.0558	-0.4342	IonFrac
INFO:  0.3376	 0.0918	 0.3700	PepLen
INFO:  0.0230	 0.0536	 0.1603	Charge1
INFO:  0.1090	 0.0586	 0.3298	Charge2
INFO: -0.1604	-0.1274	-0.5759	Charge3
INFO:  0.0000	 0.0000	 0.0000	enzN
INFO:  0.0000	 0.0000	 0.0000	enzC
INFO: -0.0195	-0.0015	 0.0231	enzInt
INFO:  0.1450	 0.0298	 0.6177	lnNumDSP
INFO: -0.1984	-0.0750	-0.2956	dM
INFO: -0.1965	-0.0327	-0.1631	absdM
INFO: -1.5185	-1.1520	-2.0882	m0
INFO: Found 54 test set PSMs with q<0.1.
INFO: Selected best-scoring PSM per scan+expMass (target-decoy competition): 104 target PSMs and 62 decoy PSMs.
INFO: Tossing out "redundant" PSMs keeping only the best scoring PSM for each unique peptide.
INFO: Calculating q values.
INFO: Final list yields 0 target peptides with q<0.1.
INFO: Calculating posterior error probabilities (PEPs).
INFO: Processing took 3.472 cpu seconds or 3 seconds wall clock time.
INFO: Elapsed time: 3.35 s
INFO: Finished crux percolator.
INFO: Return Code:0
		</pre>

		<p>
		The <span class=file>crux-output</span> directory will now contain
		eight new files:</p>
		<ol>

		<li>
		percolator.target.psms.txt &ndash; a list of peptide-spectrum matches
		(PSMs), ranked by quality,</li>

		<li>
		percolator.target.peptides.txt &ndash; a list of peptides, ranked by
		quality,</li>

		<li>
		percolator.decoy.psms.txt &ndash; a ranked list of decoy PSMs, </li>

		<li>
		percolator.decoy.peptides.txt &ndash; a ranked list of decoy peptides,</li>

		<li>
		percolator.pout.xml &ndash; a single XML output file containing all of
		the Percolator results,</li>

		<li>
		make-pin.pin.xml: an intermediate XML format file that is used by
		Percolator.</li>

		<li>
		percolator.params.txt &ndash; parameter file, and</li>

		<li>
		percolator.log.txt &ndash; log file.</li>

		</ol>
		</p>

		<p>
		  As before, you might want to sort the Percolator output files, this
		  time by the "percolator score" column:</p>


		<li>
		<div class=code><code>
		$ crux sort-by-column --column-type real --ascending T crux-output/percolator.target.psms.txt "percolator score" > crux-output/percolator.target.psms.sort.txt
		</code>
		</div>

		<p>
		The beginning of the resulting percolator.target.psms.sort.txt file will look like
		this:</p>

		<table class="searchtable">
		<td><b>file</b></td><td><b>file_idx</b></td><td><b>scan</b></td><td><b>charge</b></td><td><b>spectrum precursor m/z</b></td><td><b>spectrum neutral mass</b></td><td><b>peptide mass</b></td><td><b>percolator score</b></td><td><b>percolator rank</b></td><td><b>percolator q-value</b></td><td><b>total matches/spectrum</b></td><td><b>sequence</b></td><td><b>cleavage type</b></td><td><b>protein id</b></td><td><b>flanking aa</b></td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>118</td><td>3</td><td>1031.9407</td><td>3092.8000</td><td>3095.2366</td><td>8.6190662</td><td>1</td><td>0</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C</td><td>KD</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>26</td><td>2</td><td>692.3773</td><td>1382.7400</td><td>1382.4467</td><td>7.0481362</td><td>2</td><td>0</td><td>6</td><td>TASEFDSAIAQDK</td><td>trypsin-full-digest</td><td>YLR043C</td><td>KL</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>53</td><td>2</td><td>745.7473</td><td>1489.4800</td><td>1489.7318</td><td>6.7469611</td><td>3</td><td>0</td><td>3</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W</td><td>RN</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>62</td><td>2</td><td>745.8572</td><td>1489.7000</td><td>1489.7318</td><td>6.6489625</td><td>4</td><td>0</td><td>3</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W</td><td>RN</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>146</td><td>2</td><td>692.6772</td><td>1383.3400</td><td>1382.4467</td><td>6.6027999</td><td>5</td><td>0</td><td>5</td><td>TASEFDSAIAQDK</td><td>trypsin-full-digest</td><td>YLR043C</td><td>KL</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>131</td><td>2</td><td>745.8473</td><td>1489.6801</td><td>1489.7318</td><td>6.5627294</td><td>6</td><td>0</td><td>3</td><td>NFLETVELQVGLK</td><td>trypsin-full-digest</td><td>YGL135W</td><td>RN</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>50</td><td>2</td><td>651.2873</td><td>1300.5601</td><td>1301.4160</td><td>6.3757763</td><td>7</td><td>0</td><td>10</td><td>LDVDELGDVAQK</td><td>trypsin-full-digest</td><td>YLR043C</td><td>KN</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>42</td><td>3</td><td>1032.2606</td><td>3093.7600</td><td>3095.2366</td><td>5.8047724</td><td>8</td><td>0</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C</td><td>KD</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>90</td><td>3</td><td>1032.0006</td><td>3092.9800</td><td>3095.2366</td><td>5.4775882</td><td>9</td><td>0</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C</td><td>KD</td></tr>
		<tr>
		<td>crux-output/tide-search.target.txt</td><td>1</td><td>111</td><td>3</td><td>1033.0206</td><td>3096.0400</td><td>3095.2366</td><td>5.4364419</td><td>10</td><td>0</td><td>2</td><td>ELESAAYDHAEPVQPEDAPQDIANDELK</td><td>trypsin-full-digest</td><td>YGL009C</td><td>KD</td></tr>
		</table>

		<p>
		In this output, the PSMs are ranked by "percolator score," with higher
		scores indicating a higher quality match. The associated statistical
		confidence estimate is reported as a "percolator q-value," interpreted
		as the minimal false discovery rate threshold at which this match is
		deemed significant.  In the list above, all of the matches have
		q-values of 0, meaning that they are highly significant.
		The meanings of the remaining columns are
		described <a href="../file-formats/txt-format.html">here</a>.  Note that when you run
		Percolator on your own computer, the results may be somewhat
		different than the ones reported here.  This is because Percolator
		involves randomly subdividing the data in a cross-validation scheme
		(described in detail
		<a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-13-S16-S3">here</a>.)</p>
		</li>
		</ol>
	<!-- END CONTENT -->
		</div>
	</div>

<footer class="site-footer">
	<div id="centerfooter">
		<div class="footerimportantlinks">
			<img src="../images/linkicon.png" style="width:16px; height:16px"><h3>Important links</h3>
			<ul>
				<li><a href="../faq.html">Crux FAQ</a></li>
				<li><a href="../glossary.html">Glossary of terminology</a></li>
				<li><a href="http://scholar.google.com/citations?hl=en&user=Rw9S1HIAAAAJ">Google Scholar profile</a></li>
				<li><a href="https://github.com/crux-toolkit/crux-toolkit">GitHub</a></li>
				<li><a href="../release-notes.html">Release Notes</a></li>
				<li><a href="https://mailman1.u.washington.edu/mailman/listinfo/crux-users" title="Receive announcements of new versions">Join the mailing list</a></li>
				<li><a href="http://www.apache.org/licenses/LICENSE-2.0">Apache license</a></li>
				<li><a href="http://groups.google.com/group/crux-users">Support Board</a></li>
			</ul>
		</div>
		<div class="footerimportantlinks tutoriallinks">
			<img src="../images/tutorialicon.png" style="height:16px"><h3>Tutorials</h3>
			<ul>
				<li><a href="install.html">Installation</a></li>
				<li><a href="gettingstarted.html">Getting started with Crux</a></li>
				<li><a href="search.html">Running a simple search using Tide and Percolator</a></li>
				<li><a href="customizedsearch.html">Customization and search options</a></li>
				<li><a href="spectralcounts.html">Using spectral-counts</a></li>
			</ul>
		</div>
		<div id="footertext">
			<p>
				The original version of Crux was written by Chris Park and Aaron Klammer
				under the supervision
				of <a href="http://www.gs.washington.edu/faculty/maccoss.htm">Prof. Michael
				MacCoss</a>
				and <a href="http://noble.gs.washington.edu/~noble">Prof. William
				Stafford Noble</a> in the Department of Genome Sciences at the
				University of Washington, Seattle.  Website by <a href="http://www.yuvalboss.com/">Yuval Boss</a>. 
				<br />The complete list of contributors
				can be found <a href="../contributors.html">here</a>.
				<br />
				<br />
				Maintenance and development of Crux is funded by the <a href="https://www.nih.gov/">National Institutes of Health</a> award P41 GM103533. 
			</p>
		</div>
	</div>
</footer>
	
</body>

</html>
